extensions/functions_boolean.yaml

```diff
@@ -1,14 +1,6 @@
 %YAML 1.2
 ---
 scalar_functions:
-  -
-    name: not
-    description: The boolean not of a provided value.
-    impls:
-      - args:
-          - value: boolean
```

Comment:
I think we should just remove the second arg as opposed to removing the entire function. (It should be a single boolean argument input as opposed to two.)

Reply:
The unary not kernel was already at the end of the list

Reply:
lol

Reply:
Yes, the unary not function is already present in the end
https://github.com/substrait-io/substrait/blob/c62c1492dd0ecea04f3d053e93eedf1b79877ab2/extensions/functions_boolean.yaml#L31